web/react/stores/browser_store.jsx

```diff
@@ -25,9 +22,9 @@ class BrowserStoreClass {
         this.isLocalStorageSupported = this.isLocalStorageSupported.bind(this);
 
         var currentVersion = localStorage.getItem('local_storage_version');
-        if (currentVersion !== BROWSER_STORE_VERSION) {
+        if (currentVersion !== global.window.config.Version) {
```

Comment:
This will cause browser store deletion on a version upgrade. Probably a good thing, but the version upgrade will kill your WIP drafts.